uniflow/op/model/model_server.py

```diff
@@ -621,44 +613,38 @@ def __init__(
             import boto3
 
             super().__init__(prompt_template, model_config)
-            self._model_config = BedrockModelConfig(**self._model_config)
 
             # If user specifies profile in model config, use that profile
             if "aws_profile" in model_config:
-                aws_profile = self._model_config.aws_profile
-                session = boto3.Session(profile_name=aws_profile)
+                aws_profile = model_config["aws_profile"]
+                self.session = boto3.Session(profile_name=aws_profile)
```

Comment:
nit: `self._session` to indicate private variable?

Reply:
Sure, will fix them now